src/helper.js

```diff
@@ -0,0 +1,29 @@
+const getErrors = (givenText, givenCase) => {
+  const acceptableCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+  const errors = [];
+
+  if (!givenText) {
+    errors.push({
+      message: 'Text to convert is required. '
+        + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
```

Comment:
You can create an object with all your default error messages and use them as reference